spec/mainadditions.html

```diff
@@ -188,6 +188,40 @@ <h1>Overview of Date Objects and Definitions of Abstract Operations</h1>
 
     <p>[...]</p>
 
+    <emu-clause id="sec-temporal-getutcepochnanoseconds" type="abstract operation">
+      <h1>
+        GetUTCEpochNanoseconds (
+          _year_: an integer,
+          _month_: an integer in the inclusive interval from 1 to 12,
+          _day_: an integer in the inclusive interval from 1 to 31,
+          _hour_: an integer in the inclusive interval from 0 to 23,
+          _minute_: an integer in the inclusive interval from 0 to 59,
+          _second_: an integer in the inclusive interval from 0 to 59,
+          _millisecond_: an integer in the inclusive interval from 0 to 999,
+          _microsecond_: an integer in the inclusive interval from 0 to 999,
+          _nanosecond_: an integer in the inclusive interval from 0 to 999,
+          <ins>optional _offsetNanoseconds_: an integer in the exclusive interval from -nsPerDay to nsPerDay,</ins>
```

Comment:
Nitpick: Unfortunately ECMA-262 doesn't define what an exclusive interval is, you have to say which bound(s) it's exclusive at.
```suggestion
          <ins>optional _offsetNanoseconds_: an integer in the interval from -nsPerDay (exclusive) to nsPerDay (exclusive),</ins>
```